Updates in TPM 3.1 (#8821)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

## PARAMETERS

### -Credential

>   *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*

Reviewer: ```suggestion
  *This parameter has been deprecated from Teams Powershell Module 3.0 and later as it is no longer relevant to Microsoft Teams*.
```

>   *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.

Reviewer: ```suggestion
  *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

```

### -DomainController

>   *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*

Reviewer: ```suggestion
  *This parameter has been deprecated from Teams Powershell Module 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

>   *This parameter has been deprecated from Teams Powershell Module 3.0 and above as it is no longer relevant to Microsoft Teams*.

Reviewer: ```suggestion
  *This parameter has been deprecated from Teams PowerShell Module 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

```

### -LdapFilter
 

>   *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*

Reviewer: ```suggestion
  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

>   *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.

Reviewer: ```suggestion
  *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

```

### -OnModernServer
 

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -OnOfficeCommunicationServer
 

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

```

### -OU
 

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longet relevant to Microsoft Teams*

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above as it is no longer relevant to Microsoft Teams*.
```

```

### -SkipUserPolicies
 

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -UnassignedUser
 

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

>  *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
 *This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### CommonParameters
 

>  *These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
 *These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
